server/templates/staff/student/assignment.overview.html

```diff
@@ -0,0 +1,320 @@
+                              </div>
+                          </div>
+                          {% endif %}
+                      </li>
+                      <!-- END timeline item -->
+                      {% endfor %}
+                      <li>
+                          <i class="fa fa-clock-o bg-gray"></i>
+                      </li>
+                  </ul>
+          </div>
+          {% endfor %}
+        </div>
+        
+        <div class="row">
+            <div class="col-md-6 col-xs-12">
+                <p> Student Diff against previous commit </p>
+                {% for files in files_list %}
+                    <div class="subcontent list diff-delta">
+                        <div class="wrap">
+                            {% for filename, lines in files | dictsort %}
+                            {{ diff.diff_lines(lines) }}
+                            {% endfor %}
+                        </div>
+                    </div>
+                {% endfor %}
+            </div>
+            <div class="col-md-6 col-xs-12">
+                <p> Student Diff against previous partner commit </p>
+                {% for files in partner_files_list %} 
+                    <div class="subcontent list prev-backup">
+                      {% if not files %}
+                      <p> No previous partner commit </p>
+                      {% else %}
+                        <div class="wrap">
+                            {% for filename, lines in files | dictsort %}
+                            {{ diff.diff_lines(lines) }}
+                            {% endfor %}
+                        </div>
+                      {% endif %}
+                    </div>
+                {% endfor %}
+            </div>
+        </div>
+
+        <div class="row">
+        </div>
+
+        <!-- /.box -->
+
+        <script type="text/template" id="editor-template">
+            {{ diff.editor_template() }}
+        </script>
+
+        <!-- /.box -->
+        <!-- </section> do not close the content section -->
+
+        <!-- </body> do not close body in template-->
+        {% endblock %}
+
+{% macro stats_box(stat) %}
+    <div class="row diff-delta-box">
+        <div class="col-md-12 col-xs-1">
+            <div class="box box-default collapsed-box">
+                <div class="box-header with-border text-center">
+                    <h1 class="box-title">Diff Information</h1>
+                    <p> submitted email = {{stat.submitter}} </p>
+                    <p> commit id = {{stat.commit_id}} </p>
+                    <p> partner commit id = {{stat.partner_commit_id}} </p>
+                    <p> question = {{stat.question}} </p>
+                    <p> time = {{stat.time}} </p>
+                    <p> passed = {{stat.passed}} </p>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endmacro %}
+
+
+{% block page_js %}
+<script type="text/javascript">
+    // show only diff 1
+    $('.timeline').hide()
```

Comment:
Could we have this all be hidden by default and then show this so we don't see a ton of content on the screen and a flash where it disappears